conf/openrefine.l4j.ini

```diff
@@ -8,6 +8,16 @@
 
 # Use system defined HTTP proxies
 -Djava.net.useSystemProxies=true
+# You should probably define the properties below, if they are not already 
+# defined in the environment variable JAVA_TOOL_OPTIONS.
+# Refer to the Java documentation (for example, at
+# https://docs.oracle.com/javase/8/docs/api/java/net/doc-files/net-properties.html
+# for the precise meaning of each option)
+#-Dhttp.proxyHost=myproxy.mydomain.org
+#-Dhttp.proxyPort=8080
+#-Dhttps.proxyHost=myproxy.mydomain.org
+#-Dhttps.proxyPort=8080
+#-Dhttp.nonProxyHosts="*.mydomain.org|localhost|127.*"
```

Comment:
It would also be worth adding this to `refine.ini` (for Linux) using the `JAVA_OPTIONS=-Dhttp.proxy....` syntax documented at https://openrefine.org/docs/manual/running#starting-with-modifications
It's really a mess though, we ought to have something uniform across platforms… 

If that feels too fiddly to add to all platforms now, I would be okay with waiting for https://github.com/OpenRefine/OpenRefine/issues/3057 to sort it out.